tensorflow/lite/examples/label_image/label_image.cc

```diff
@@ -85,9 +85,21 @@ TfLiteDelegatePtrMap GetDelegates(Settings* s) {
     if (!delegate) {
       LOG(INFO) << "NNAPI acceleration is unsupported on this platform.";
     } else {
-      delegates.emplace("NNAPI", evaluation::CreateNNAPIDelegate());
+      delegates.emplace("NNAPI", std::move(delegate));
     }
   }
+
+  if (s->hexagon_delegate) {
+    const std::string libhexagon_path("/data/local/tmp");
+    auto delegate = evaluation::CreateHexagonDelegate(libhexagon_path);
```

Comment:
The function takes another param. please fix.

Reply:
oops, will do it. my original patch was made before the `evaluation::CreateHexagonDelegate()` change, 1fac666679eb4353513b226060c844aa5a1d78ec.

Reply:
done